web/controllers/subjects_controller.ex

```diff
@@ -0,0 +1,77 @@
+defmodule ActiveMonitoring.SubjectsController do
+  use ActiveMonitoring.Web, :controller
+
+  alias ActiveMonitoring.{
+    Campaign,
+    ChangesetView,
+    Repo,
+    Subject,
+  }
+
+  require Logger
+
+  def index(conn, %{"campaigns_id" => campaign_id} = params) do
+    limit = Map.get(params, "limit", "50") |> String.to_integer
+    page = Map.get(params, "page", "1") |> String.to_integer
+
+    limit = limit_up_to_50(limit)
```

Comment:
An alternative way to this is to continue the pipeline of line 14 passing max(50):

`limit = Map.get(params, "limit", "50") |> String.to_integer |> max(50)`

Reply:
The idea was to handle invalid params - and default the page size.

It wasn't working, anyways, so we've updated that and added some specs to test those cases 👍